Sources/Basic/FileSystem.swift

```diff
@@ -632,6 +647,11 @@ public class InMemoryFileSystem: FileSystem {
     public var currentWorkingDirectory: AbsolutePath? {
         return AbsolutePath("/")
     }
+    
+    /// Virtualized current home directory.
+    public var homeDirectory: AbsolutePath {
+        return AbsolutePath("/home/user")
```

Comment:
This value is defined for mock testing. But I think it should not be contained here...

Reply:
Lets just return "/" for now.